components/Navbar/NavbarDropdownComponent.tsx

```diff
@@ -0,0 +1,125 @@
+"use client";
+import React, { useState } from "react";
+import { Prisma } from "@/lib/prisma";
+
+export type NavbarDropdownProps = Omit<
+  Prisma.NavbarDropdownsGetPayload<{}>,
+  "items"
+> & {
+  items: {
+    title: string;
+    groups_with: string;
+    url: string;
+  }[];
+};
+
+type Item = {
+  title: string;
+  groups_with: string;
+  url: string;
+};
+
+export function NavbarDropdown({ title, items }: NavbarDropdownProps) {
+  const [open, setOpen] = useState(false);
+  const changeState = () => {
+    setOpen(!open);
+  };
+
+  const groupedItems: Item[][] = items.reduce((groups, item) => {
+    if (
+      groups.length === 0 ||
+      groups[groups.length - 1][0].groups_with !== item.groups_with
+    ) {
+      groups.push([item]);
+    } else {
+      groups[groups.length - 1].push(item);
+    }
+    return groups;
+  }, [] as Item[][]);
+
+  return (
+    <>
+      {/* MOBILE */}
+      <div className="md:hidden flex flex-col gap-1">
+        <button
+          onClick={changeState}
+          className="text-brand-yellow text-2xl font-rockingsoda bg-primary  w-full p-3 flex justify-center"
+        >
+          {title}
+        </button>
+        {open && (
+          <div className="pl-2 flex gap-1 flex-col">
+            {groupedItems.map((items) => (
+              <div className="bg-primary w-full p-3 flex flex-col justify-center">
+                <div className="flex w-full gap-2 items-center">
+                  {items[0].groups_with && (
+                    <>
+                      <span className="text-[20px] font-rockingsoda">
+                        {items[0].groups_with}
+                      </span>
+                    </>
+                  )}
+                </div>
+                <div className="w-full h-[1px] bg-brand-yellow"></div>
+                <div className="flex-col flex">
+                  {items.map((item) => (
+                    <a
+                      className="text-center py-2"
+                      href={item.url || undefined}
+                    >
+                      {item.title}
+                    </a>
+                  ))}
+                </div>
+              </div>
+            ))}
+          </div>
+        )}
+      </div>
+      {/* DESKTOP */}
+      <div className="hidden md:block">
+        <button
+          onClick={changeState}
+          className="text-black text-2xl font-rockingsoda"
+        >
+          {title}
+        </button>
+        {open && (
+          <div className="z-30 min-w-36 translate-y-4 drop-shadow-[0px_0px_8px_#000000] absolute items-center bg-white flex flex-col text-black font-rockingsoda text-2xl rounded-xl p-4 border-black b-2">
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              viewBox="0 0 52.36 24.43"
+              className=" absolute w-10 h-10 -translate-x-8 -translate-y-[43.4px] z-20 pointer-events-none"
+            >
+              <path
+                className="fill-white stroke-none"
```

Comment:
I'd avoid using tailwind in SVGs, since good practices are to have SVGs self contained, and not dependent on the styles of the document. In this case you can use `fill="white"` and `stroke="none"`.